copy/copy.go

```diff
@@ -83,10 +86,23 @@ func (d *digestingReader) Read(p []byte) (int, error) {
 type Options struct {
 	RemoveSignatures bool   // Remove any pre-existing signatures. SignBy will still add a new signature.
 	SignBy           string // If non-empty, asks for a signature to be added during the copy, and specifies a key ID, as accepted by signature.NewGPGSigningMechanism().SignDockerManifest(),
+	ReportWriter     io.Writer
```

Comment:
(`io.Writer` is enough as far as types go; I was thinking about the semantics of the output only.)

Right, this works just fine for now and if anyone wanted a truly stable interface, it would need to be quite different. Let’s not worry about it then :) Feel free to leave this as is.